tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -437,6 +437,12 @@ public:
    /// When writing a variable size array through Snapshot, it is required that the column indicating its size is also
    /// written out and it appears before the array in the columnList.
    ///
+   /// \attention In multi-thread runs (i.e. when EnableImplicitMT() has been called) threads will loop over clusters of
+   /// entries in undefined order, so Snapshot will produce outputs in which (clusters of) entries will be shuffled with
+   /// respect to the input TTree. Using such "shuffled" TTrees as friends of the original trees would result in wrong
+   /// associations between entries in the main TTree and entries in the "shuffled" friend. Recent ROOT versions will
```

Comment:
Is it possible to state which specific ROOT version changed this behavior?

Reply:
absolutely